src/tactic/ui/widget/gear_menu_wdg.py

```diff
@@ -79,8 +79,30 @@ def get_args_keys(my):
             "embedded_table": "boolean to show if it is part of an embedded table"
         }
 
+    def get_access_keys_dict(my):
 
-    def add_custom_menus( my, menus, menu_idx_map, custom_menus ):
+        from tactic.ui.startup import GearMenuSecurityWdg
+        get_menu = GearMenuSecurityWdg.get_menu()

```

Comment:
it should be called whatever name you have decided in the other file (not get_menu())